Jenkins Operator on Openshift.md

```diff
@@ -0,0 +1,185 @@
+Jenkins Operator on Openshift

```

Comment:
we probably need to put it on the website, [somewhere in `website/content/en/docs`](https://github.com/jenkinsci/kubernetes-operator/tree/master/website/content/en/docs)

Reply:
Added to the Getting Started Section.